src/x509/clu_config.c

```diff
@@ -373,9 +381,9 @@ int wolfCLU_setExtensions(WOLFSSL_X509* x509, WOLFSSL_CONF* conf, char* sect)
     }
 
     current = wolfSSL_NCONF_get_string(conf, sect, "basicConstraints");
-    if (current != NULL) {
-        wolfCLU_parseExtension(x509, current, NID_basic_constraints, &idx);
-    }
+
+    /* set basic constraint to default CA:TRUE if not specified */
+    wolfCLU_parseExtension(x509, current, NID_basic_constraints, &idx);
```

Comment:
default should be false for basic constraint, there may be some special cases like self signed certificates. Please double check this behavior.

Reply:
Double checked, looks to be TRUE for openSSL 1.1.1f

Reply:
That was wrong. Should be default only for `req -x509` now